MODAK/test/test_jobfile_generator.py

```diff
@@ -0,0 +1,169 @@
+import unittest
+import json
+import tempfile
+import jobfile_generator
+
+TEST_STRING=b"""## OPTSCRIPT here ##"""
+
+class test_mapper(unittest.TestCase):
+    def setUp(self):
```

Comment:
teardown missing?

Reply:
I don't think we need it. Everything is in temporary files which are removed as soon as they're closed.

Reply:
Ok, so they will be garbage collected at some point 😉 ... because I didn't find the `.close()`.